code/modules/client/preference_setup/loadout/loadout_accessories.dm

```diff
@@ -256,6 +294,142 @@
+	display_name = "shoulder sleeve patch (colourable)"
 	path = /obj/item/clothing/accessory/sleevepatch
 	flags = GEAR_HAS_NAME_SELECTION | GEAR_HAS_DESC_SELECTION | GEAR_HAS_COLOR_SELECTION
+
+/datum/gear/accessory/namepin
+	display_name = "pin tag (colourable)"
+	path = /obj/item/clothing/accessory/badge/namepin
+	flags = GEAR_HAS_NAME_SELECTION | GEAR_HAS_DESC_SELECTION | GEAR_HAS_COLOR_SELECTION
+
+/datum/gear/accessory/asym
+	display_name = "asymmetric jacket (selection)"
+	path = /obj/item/clothing/accessory/asymmetric
+
+/datum/gear/accessory/asym/New()
+	..()
+	var/asyms = list()
+	asyms["blue asymmetric jacket"] = /obj/item/clothing/accessory/asymmetric
+	asyms["purple asymmetric jacket"] = /obj/item/clothing/accessory/asymmetric/purple
+	asyms["green asymmetric jacket"] = /obj/item/clothing/accessory/asymmetric/green
+	gear_tweaks += new/datum/gear_tweak/path(asyms)
+
+/datum/gear/accessory/sash
+	display_name = "sash (colourable)"
+	path = /obj/item/clothing/accessory/sash
+	flags = GEAR_HAS_NAME_SELECTION | GEAR_HAS_DESC_SELECTION | GEAR_HAS_COLOR_SELECTION
+
+/datum/gear/accessory/cowledvest
+	display_name = "cowled vest"
+	path = /obj/item/clothing/accessory/cowled_vest
+
+/datum/gear/accessory/halfcape
+	display_name = "half cape"
+	path = /obj/item/clothing/accessory/halfcape
+
+/datum/gear/accessory/fullcape
+	display_name = "full cape"
+	path = /obj/item/clothing/accessory/fullcape
+
+/datum/gear/accessory/bracelet
+	display_name = "bracelet (colourable)"
+	path = /obj/item/clothing/accessory/bracelet
+	flags = GEAR_HAS_NAME_SELECTION | GEAR_HAS_DESC_SELECTION | GEAR_HAS_COLOR_SELECTION
+
+/datum/gear/accessory/bracelet_material
+	display_name = "bracelet (selection)"
+	path = /obj/item/clothing/accessory/bracelet/material/plastic
+
+/datum/gear/accessory/bracelet_material/New()
+	..()
+	var/bracelettype = list()
+	bracelettype["steel bracelet"] = /obj/item/clothing/accessory/bracelet/material/steel
+	bracelettype["iron bracelet"] = /obj/item/clothing/accessory/bracelet/material/iron
+	bracelettype["silver bracelet"] = /obj/item/clothing/accessory/bracelet/material/silver
+	bracelettype["gold bracelet"] = /obj/item/clothing/accessory/bracelet/material/gold
+	bracelettype["platinum bracelet"] = /obj/item/clothing/accessory/bracelet/material/platinum
+	bracelettype["glass bracelet"] = /obj/item/clothing/accessory/bracelet/material/glass
+	bracelettype["wood bracelet"] = /obj/item/clothing/accessory/bracelet/material/wood
+	bracelettype["plastic bracelet"] = /obj/item/clothing/accessory/bracelet/material/plastic
+	gear_tweaks += new/datum/gear_tweak/path(bracelettype)
+
+/datum/gear/accessory/bracelet_friendship
+	display_name = "bracelet, friendship"
+	path = /obj/item/clothing/accessory/bracelet/friendship
+
+/datum/gear/accessory/suitvest
+	display_name = "suit vest"
+	path = /obj/item/clothing/accessory/suitvest
+
+/datum/gear/accessory/roles/poncho/cloak/hos
+	display_name = "cloak, head of security"
```

Comment:
Remove all cloaks. We removed cloaks a while ago due to their conflict with the station and the setting's aesthetics.